docker/reset.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
```

Comment:
Since this operation could be pretty destructive, I would recommend adding a confirmation to the user:

```suggestion

echo "WARNING: This will remove all containers and their data. This action cannot be undone!"
read -p "Are you sure you want to proceed? (y/N) " -n 1 -r
echo    # Move to a new line
if [[ ! $REPLY =~ ^[Yy]$ ]]
then
    echo "Operation cancelled."
    exit 1
fi

```